### ISSUE TITLE
How to style custom buttons in the menu bar?

### ISSUE THREAD

**Issue by jtuchel** [josdejong/svelte-jsoneditor#169]
I tried to setup a "download" button that should appear in the menu bar

```ts
  const downloadButton: MenuButtonItem = {
    onClick: saveDocumentToDisk,
    // icon: {}, // download icon
    text: "Save to disk",
    title: "Save this document to disk",
    className: "jse-button",
  };
```

Unfortunately it looks like

![image](https://user-images.githubusercontent.com/92626738/196715643-dd9bbfeb-e243-4422-9792-dd4055f517fe.png)

I want it to look like the original one ( taken from here https://jsoneditoronline.org/ )

![image](https://user-images.githubusercontent.com/92626738/196716218-0c8a4ffc-4b46-496c-8f08-d5495231af63.png)

Would you mind telling me what's wrong/missing? Is there a public repository for **jsoneditoronline** I can have a look at?

**It would be fine for me if there is no text, just the icon as a button!**

**Comment by jtuchel** [comment]
@josdejong thanks for your reply. I installed FontAwesome for my Vue app and I'm trying to reproduce your linked Svelte example. I started with this

```ts
<script setup lang="ts">
import type { FontAwesomeIcon } from "vanilla-jsoneditor";
import { faCopy } from '@fortawesome/free-regular-svg-icons';

const myIcon: FontAwesomeIcon = {
  prefix: faCopy.prefix,
  iconName: faCopy.iconName,
  icon: faCopy.icon // this is not possible
};
</script>
```

I can't use `faCopy.icon` because of

> TS2322: Type '[number, number, string[], string, IconPathData]' is not assignable to type '[number, number, (string | number)[], string, string]'.   Type at position 4 in source is not compatible with type at position 4 in target.     Type 'IconPathData' is not assignable to type 'string'.       Type 'string[]' is not assignable to type 'string'.  index.d.ts(408, 5): The expected type comes from property 'icon' which is declared here on type 'FontAwesomeIcon'

`faCopy` is not of type `FontAwesomeIcon`, its type is `IconDefinition`. Did I miss something?

**Comment by josdejong** [comment]
I've fixed the issue via https://github.com/josdejong/svelte-jsoneditor/commit/9d693f94ebeffa187a2f3ab8f85998b987be8b94 (not yet published)

**Comment by jtuchel** [comment]
> I've fixed the issue via [9d693f9](https://github.com/josdejong/svelte-jsoneditor/commit/9d693f94ebeffa187a2f3ab8f85998b987be8b94) (not yet published)

Thanks for your efforts :) Do you have any plans when this will be released?

**Comment by josdejong** [comment]
👍 I've published the fix now in `v0.8.0`.

**Comment by jtuchel** [comment]
Working perfectly fine :)
